.gitignore

```diff
@@ -1,3 +1,5 @@
+.DS_Store
+
```

Comment:
Don't push this into the master. Use git exclude file, see https://www.jerriepelser.com/blog/exlude-git-files-locally/

Reply:
got it, corect